services/scabbard/libscabbard/src/service/factory.rs

```diff
@@ -156,15 +156,25 @@ impl ServiceArgValidator for ScabbardArgValidator {
         let admin_keys_str = args
             .get("admin_keys")
             .ok_or_else(|| ServiceArgValidationError("admin_keys argument not provided".into()))?;
-        let admin_keys: Vec<String> = serde_json::from_str(admin_keys_str).map_err(|err| {
```

Comment:
Typo in commit message: s/facotry/factory

Reply:
fixed